www/js/intro.js

```diff
@@ -117,7 +117,14 @@ angular.module('emission.intro', ['emission.splash.startprefs',
     // this is not a promise, so we don't need to use .then
     StartPrefs.markIntroDone();
```

Comment:
can you also, either here, or in `markIntroDone()`, set the user channel to `megumi-brt`?
Since you are using a shared server, that will let me figure out which users are for your study.
Similar to:
https://github.com/e-mission/e-mission-phone/blob/urap-2017-emotion/www/js/intro.js#L90

Reply:
alternatively, you can submit a pull request to master that updates the user client to the stored channel, which is a generic solution and will not require each channel to make this change